example/notebooks/deletion_insertion_metric_example.ipynb

```diff
@@ -352,6 +312,7 @@
     "sample_batch = next(iter(val_dataloader))\n",
     "# iterate over all samples in batch\n",
     "sample, label = sample_batch[0][0], sample_batch[1][0]\n",
+    "sample = sample.to(device)\n",
     "# add batch size dimension to the data sample\n",
     "input_data = sample.reshape(1, sample.shape[0], sample.shape[1], sample.shape[2]).to(device)\n",
     "category_name = categories[label.item()]\n",
```

Comment:
I can't directly place this comment in the line I am referring to because it was not changed. Currently in the cell below you have magic strings to refer to the explainer. Let's make it variable outside of with-clause and reuse it in multiple places. You can access explainer name by accessing `.name` property of `CVClassificationExplainers` class object.